crates/humanode-runtime/Cargo.toml

```diff
@@ -91,7 +97,10 @@ static_assertions = { version = "1.1.0", default-features = false }
 crypto-utils = { version = "0.1", path = "../crypto-utils" }
 eip712-common-test-utils = { version = "0.1", path = "../eip712-common-test-utils" }
 
+ethereum = "0.14"
```

Comment:
Maybe use `fp-ethereum` instead?

Reply:
Unfortunately, `fp-evm` doesn't expose all types required for test. For example, the next one structure can be obtained from `ethereum` itself:

```
pallet_ethereum::Transaction::Legacy(ethereum::LegacyTransaction {
			nonce: 0.into(),
			gas_price: gas_price.into(),
			gas_limit: gas_limit.into(),
			action: ethereum::TransactionAction::Call(
				hex_literal::hex!("727fc6a68321b754475c668a6abfb6e9e71c169a").into(),
			),
			value: U256::from(10) * 1_000_000_000 * 1_000_000_000,
			input: hex_literal::hex!("a9059cbb000000000213ed0f886efd100b67c7e4ec0a85a7d20dc971600000000000000000000015af1d78b58c4000").into(),
			signature: ethereum::TransactionSignature::new(38, hex_literal::hex!("be67e0a07db67da8d446f76add590e54b6e92cb6b8f9835aeb67540579a27717").into(), hex_literal::hex!("2d690516512020171c1ec870f6ff45398cc8609250326be89915fb538e7bd718").into()).unwrap(),
		});
```

Reply:
Maybe it doesn't, nvm